openmetadata-ui/src/main/resources/ui/src/utils/TableUtils.tsx

```diff
@@ -753,7 +753,14 @@ export const getTableDetailPageBaseTabs = ({
 }: TableDetailPageTabProps): TabProps[] => {
   return [
     {
-      label: <TabsLabel id={EntityTabs.SCHEMA} name={t('label.schema')} />,
+      label:  (
+        <TabsLabel
+          count={tableDetails?.columns.length}
+          id={EntityTabs.SCHEMA}
+          isActive={activeTab === EntityTabs.SCHEMA}
+          name={t('label.schema')}
+        />
+      ),
```

Comment:
We also have schema in other entity, I think we should also add similar behaviour there!